tests/functional/sso/__init__.py

```diff
@@ -80,6 +80,63 @@ def tearDown(self):
         self.uuid_patch.stop()
         self.token_cache_dir_patch.stop()
 
+    def add_oidc_device_responses(self, access_token,
```

Comment:
Just moving this into the base so another child can use it, same contents as before.